fastlane/lib/fastlane/actions/appledoc.rb

```diff
@@ -180,8 +180,8 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :warnings, env_name: "FL_APPLEDOC_WARNINGS", description: "Documentation generation warnings", is_string: true, optional: true),
 
           # MISCELLANEOUS
-          FastlaneCore::ConfigItem.new(key: :logformat, env_name: "FL_APPLEDOC_LOGFORMAT", description: "Log format [0-3]", is_string: false, optional: true),
-          FastlaneCore::ConfigItem.new(key: :verbose, env_name: "FL_APPLEDOC_VERBOSE", description: "Log verbosity level [0-6,xcode]", is_string: false, optional: true)
+          FastlaneCore::ConfigItem.new(key: :logformat, env_name: "FL_APPLEDOC_LOGFORMAT", description: "Log format [0-3]", type: Integer, optional: true),
```

Comment:
If this change is needed because it breaks compatibility with the previous parsing, plugins will be affected as well and then will break them.

Reply:
Thank you for your review and reverted these changes to ensure that this PR will not break the previous compatibility!
Requesting you to do a re-review! 😇